vela-templates/definitions/internal/component/config-helm-repository.cue

```diff
@@ -0,0 +1,41 @@
+"config-helm-repository": {
+	type: "component"
+	annotations: {
+		"alias.config.oam.dev": "Helm Repository"
+	}
+	labels: {
+		"catalog.config.oam.dev":       "velacore-config"
+		"type.config.oam.dev":          "helm-repository"
+		"multi-cluster.config.oam.dev": "true"
```

Comment:
Please add `ui-hidden： "true"`

Reply:
Sure. I will also move it to addon fluxcd as what you did for dex addon.

Reply:
remove this definition from the core. I have already added it to the flux addon. https://github.com/oam-dev/catalog/pull/304